client/src/css/Ellie.css

```diff
@@ -0,0 +1,62 @@
+@import url('https://fonts.googleapis.com/css2?family=Cabin&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Comfortaa:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+.welcome {
+  margin: 5%;
+  align-items: center;
+  justify-content: left;
+}
+
+#home-picture {
+  width: 200px;
+  height: 200x;
+}
+
+.web-picture {
+  width: 200px;
+  height: 200px;
+  padding-left: 5%;
+}
+
+.picture {
+  order: 1;
+}
+
+.text {
+  order: 2;
+  font-size: 20px;
+  font-family: 'Poppins', sans-serif;
+  text-align: left;
+}
```

Comment:
These classNames are a big general. Name them something more specific because other pages may also have text and pictures and if they have the same classNames, you'll affect their page too. You can even just name them .ellie-text or .ellie-picture.